mpact/views.py

```diff
@@ -161,6 +161,28 @@ def post(self, request):
         return Response(result[DATA], status=result[STATUS])
 
 
+class StudyParticipants(APIView):
```

Comment:
Are you thinking some additional buttons to upload / download these like the messages? I'm thinking we cannot do it through admin since we have to trigger the `import_participants` code.

Reply:
yeah, we'll need a UI somewhere.